keps/prod-readiness/sig-network/4444.yaml

```diff
@@ -3,3 +3,5 @@ alpha:
   approver: "@johnbelamaric"
 beta:
   approver: "@johnbelamaric"
+stable:
```

Comment:
Given we effectively want to rename PreferClose to PreferSameZone: https://github.com/kubernetes/enhancements/pull/5140
should we graduate the name we don't like to GA?

I agree there is strong argument about not breaking existing Beta customers, but given we already know we want to change that and the KEP adding a new name for the same behavior (PreferSameZone) is already out - should we revisit this decision?
Or are we saying that given we support the bahvior itself, the cost of supporting an alias for it is so negligible that the gain of not breaking users outweighs it?

@thockin @danwinship - what are your thoughts here?

Reply:
In theory: we could move it from beta and on by default to deprecated and on by default, but that's not quite capturing what we want to happen. We want the _mechanism_ to be generally available, and we're not telling people to avoid using that mechanism.